networking/hardware_networks/uninstalling-sriov-operator.adoc

```diff
@@ -0,0 +1,57 @@
+[id="uninstalling-sriov-operator"]
+= Uninstalling the SR-IOV Network Operator
+include::modules/common-attributes.adoc[]
+:context: uninstalling-sr-iov-operator
+
+toc::[]
+
+To uninstall the SR-IOV Network Operator, you must first delete any running SR-IOV workloads. You must also delete the webhooks after uninstalling the operator.
+
+.Prerequisites
+
+* You must have access to an {product-title} cluster using an account with `cluster-admin` permissions.
+* You have the SR-IOV Network Operator installed.
+
+.Procedure
+
+. Delete all SR-IOV custom resources (CRs):
++
+[source,terminal]
+----
+$ oc delete sriovnetwork -n openshift-sriov-network-operator --all
+$ oc delete sriovnetworknodepolicy -n openshift-sriov-network-operator --all
+$ oc delete sriovibnetwork -n openshift-sriov-network-operator --all
+----
+
+. Follow the instructions on xref:../../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster[Deleting operators from a cluster] to remove the SR-IOV Network Operator from your cluster.
+
+. The SR-IOV resource definitions (CRDs) remain in the cluster after the SR-IOV Network Operator is uninstalled. Delete the remaining SR-IOV CRDs:
++
+[source,terminal]
+----
+$ oc get crd -o name | grep sriov | xargs oc delete
+----
+
+. Delete the SR-IOV webhooks:
++
+[source,terminal]
+----
+$ oc delete mutatingwebhookconfigurations network-resources-injector-config
+$ oc delete MutatingWebhookConfiguration sriov-operator-webhook-config
+$ oc delete ValidatingWebhookConfiguration sriov-operator-webhook-config
+----
+
+. Delete references to stray finalizers from the SriovNetworks:
```

Comment:
"Stray" isn't used anywhere else in the docs collection, and our style guide suggests avoiding unique terms when possible. I suggest:
```suggestion
. Delete references to unused finalizers from the SriovNetworks:
```